CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+## [Unreleased]
+<details>
+  <summary>
+    Changes that have landed in master but are not yet released.
+    Click to see more.
+  </summary>
+
+### New JS Environment Requirements
+
+ * **React 16 depends on the collection types [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) and [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), as well as [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame).** If you support older browsers and devices which may not yet provide these natively (eg <IE11), [you may want to include a polyfill](https://gist.github.com/gaearon/9a4d54653ae9c50af6c54b4e0e56b583).
+
+### New Features
+* Components can now return arrays and strings from `render`. (Docs coming soon!)
+* **Improved error handling with introduction of "error boundaries".** [Error boundaries](https://facebook.github.io/react/blog/2017/07/26/error-handling-in-react-16.html) are React components that catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI instead of the component tree that crashed.
+* `ReactDOM.createPortal()` is official and stable
```

Comment:
It's not clear what it is. We generally try to make changelog easy to read for somehow who's not very deeply familiar with React. For example this could say: "First-class support for declaratively rendering a subtree into another DOM node with `ReactDOM.createPortal()`."

Reply:
(I don't think it's important to give the PR number here since the only thing my PR did was rename a method. But people reading this probably aren't familiar with the whole idea itself. IMO it's only useful to link to PRs if they show actual implementation. For this case let's just not link to it.)

Reply:
Yea - I was trying to find something to link to since we don't have docs yet but I can just say 'Docs coming soon' like in one of the previous points.